docs/tutorial/fdfitting.rst

```diff
@@ -0,0 +1,343 @@
+
+Frequently, global fits have better statistical properties than fitting the data separately
+as more information is available to infer parameters shared between the various models.
+
+Plotting the data
+-----------------
+
+Fits can be plotted using the built-in plot functionality::
+    
+    odijk_fit.plot()
+    plt.ylabel('Force [pN]')
+    plt.xlabel('Distance [$\\mu$M]');
+
+However, sometimes more fine grained control over the plots is required. Let's say we want to plot
+the model over the range 2.0 to 5.0 for the conditions from `Control` and `RecA`. We can do this by
+calling plot on the model directly::
+
+    odijk_with_offsets.plot(odijk_fit['Control'], np.arange(2.0, 5.0, .01), fmt='k--')
+    odijk_with_offsets.plot(odijk_fit['RecA'], np.arange(2.0, 5.0, .01), fmt='k--')
+
+Note how we use the square brackets to select the parameters belonging to condition 1 and 2 using
+the data set names. This collects the parameters relevant for that particular experimental condition.
+
+It is also possible to obtain simulations from the model directly. We can do this by calling the 
+model with values for the independent variable (here denoted as distance) and the parameters 
+required to simulate the model. Again, we obtain these parameters by grabbing them from our fit
+object using the data handles::
+
+    distance = np.arange(2.0, 5.0, .01)
+    simulation_result = odijk_with_offsets(distance, odijk_fit['Control'])
+
+Multiple models
+---------------
+
+When working with multiple models things can get a little more complicated. Let's say we have
+two models, `model1` and `model2` and we want to fit both in a global fit. Constructing the
+`FdFit` is easy::
+
+    model1 = pylake.inverted_odijk("DNA")
+    model2 = (pylake.odijk("DNA") + pylake.odijk("protein")).invert()
+    fit = pylake.FdFit(model1, model2)
+
+But then the question arises, how do we add data to each model? Well, the trick is in the
+assignments to `model1` and `model2`. We can use these now to add data to each model as
+follows::
+
+    fit[model1].add_data("data for model 1", forces_1, distances_1)
+    fit[model2].add_data("data for model 2", forces_2, distances_2)
+
+See how we used the model handles? They are used to let the `FdFit` know where to add
+each data set. You can add as many data sets as you want to both models, and fit it all
+at once.
+
+Also accessing the model parameters for a specific dataset is a little more complicated
+in this setting. If we for example want to plot "data for model 1", we'd have to invoke::
+
+    model1.plot(fit[model1]['data for model 1'], np.arange(2.0, 5.0, .01), fmt='k--')
+
+Note how we are now forced to index the model first using the square brackets, and only
+then access the data set by name. An unfortunate necessity when it comes to multi-model
+curve fitting.
+
+Global fits versus single fits
+------------------------------
+
+The `Fit` object manages a fit. To illustrate its use, and how a global fit differs from a
```

Comment:
Correct. That one must have slipped through the cracks, thanks.

`Fit` still exists. `FdFit` is a specialized one that has a custom `add_data` that makes sure the order is always `f`, `d` irrespective of whether you have a forward or inverse model (since that lead to confusions otherwise).

In addition, the specialization is nice to have for when we actually couple this more directly to the `FdCurve` class, as we can then keep the `Fd`-specific stuff contained in its own class. I could see reusing some of this more general infrastructure for other parameter estimation problems for instance.